sources/sections-a4-en/03-base-units.adoc

```diff
@@ -1,20 +1,24 @@
-== Part 3. Historical perspective on the base units(((base unit(s))))
+== Historical perspective on the base units(((base unit(s))))
 
+[%unnumbered]
```

Comment:
All A4 sections are to be numbered, so let's remove `[%unnumbered]`. Only A1 sections (the quoted resolutions/decisions) are to be unnumbered.

Reply:
Done.